common/page/Footer.tsx

```diff
@@ -101,7 +101,7 @@ export function Footer() {
         </NavigationItem>
       </NavigationList>
       <LegalInfo>
-        &copy; 2020 The Discohook Authors. Discohook is not affiliated with
+        &copy; 2020-2021 The Discohook Authors. Discohook is not affiliated with
```

Comment:
I prefer a year range not be used
```suggestion
        &copy; 2021 The Discohook Authors. Discohook is not affiliated with
```